### ISSUE TITLE
Parallelization Bug: only the last completed test run block's tests are reported .test_result bundle 

### ISSUE THREAD

**Issue by ishammoho** [lyndsey-ferguson/fastlane-plugin-test_center#107]
### New Issue Checklist

- [ ] Updated `fastlane-plugin-test_center` to the latest version
- [ ] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [ ] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [ ] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

### Issue Description
Only the last completed test run block's tests are reported .test_result bundle when running tests using parallelization**

<details>
  <pre>[INSERT OUTPUT HERE]</pre>
</details>

### Environment

<details>
  <pre>[INSERT OUTPUT HERE]</pre>
</details>

**Comment by lyndsey-ferguson** [comment]
@ishammoho please test [Add support to not collate report files ](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/pull/108) to confirm that this allows you to use the intermediate test_bundles.

**Comment by lyndsey-ferguson** [comment]
@szymon-kazmierczak, you too please ^

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.8.0.parallelizing.beta.4](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.8.0.parallelizing.beta.4)

**Comment by ishammoho** [comment]
when I ask multi_scan to collate the results this is still happening. However when I set collate: false to generate report using each batch reporting part is fine. However if there are re-tried tests the build shows as failed.

**Comment by lyndsey-ferguson** [comment]
@ishammoho thank you for the information. Yes, I can imagine that the build will show as failed as the uncollated test results will have failed tests in them. Collating will remove failed tests *if* those same tests passed in a later run.

With collation, I imagine that the way that they are being collated is not compatible with `xchtmlreport`. If you can create a separate ticket for that issue, I can look at it at a later date.

**Comment by jineshqa** [comment]
@lyndsey-ferguson as you know xchtmlreport makes report easily consumable with screenshot of failure on the side. Could you look into fixing this issue or suggest a workaround?

**Comment by lyndsey-ferguson** [comment]
Can you create a ticket with a sample project, Fastfile, what you're expecting in the collated report and what you're getting instead? Thanks!

**Comment by jineshqa** [comment]
I can create a ticket but not a sample project where I can reproduce the exact error. I was running tests in parallel, 4 sims. Report showed 9 failures in total but xchtmlreport shows only 3.

Should I create a ticket with the info and attach my result bundle?

**Comment by lyndsey-ferguson** [comment]
Yes please. The main thing is that I need to know what you're expecting and how the collation is not meeting those expectations.
